extensions/sql-database-projects/package.nls.json

```diff
@@ -0,0 +1,7 @@
+{
+	"sqlDatabaseProjects.displayName": "Database Projects",
+	"sqlDatabaseProjects.description": "View and run previously executed queries",
```

Comment:
"View and run previously executed queries" - is it..?

Reply:
Nope; missed copy-pasta